docs/changelog/biweekly-report/2022-12-16-biweekly-report.md

```diff
@@ -0,0 +1,110 @@
+
+It's only been three weeks and the community has already received lots of attention and powerful support from developers scattered around the world. Their talent and passion took us by surprise and strengthened our belief in open source.
+
+*Biweekly Reports* will be a series of quick reports that update our progress regularly. We hope, in this way, members and friends could understand what the project is doing and how they would participate in the GreptimeDB community.
+
+Here are also some good examples of our issue pools. They would be perfect for newcomers, so please have faith in yourself and join us right away!
+
+Join us at [github.com](https://github.com/GreptimeTeam/greptimedb)
+
+## Contributors list: (In alphabetical order)
+
+[@azhsmesos](https://github.com/azhsmesos) [@DiamondMofeng](https://github.com/DiamondMofeng) [@dongxuwang](https://github.com/dongxuwang)
+
+[@e1ijah1](https://github.com/e1ijah1) [@francis-du](https://github.com/francis-du) [@lbt05](https://github.com/lbt05)
+
+[@lizhemingi](https://github.com/lizhemingi) [@morigs](https://github.com/morigs) [@SSebo](https://github.com/SSebo) [@Xuanwo](https://github.com/Xuanwo)
+
+A big THANK YOU for the generous and brilliant contributions! It is people like you who are making GreptimeDB a great product. Let's keep up the good work!
+
+Join us at [github.com](https://github.com/GreptimeTeam/greptimedb)
+
+## Good first issue
+
+### Issue #667 (Closed)
+
+Link：https://github.com/GreptimeTeam/greptimedb/issues/667
+
+Issue description:
+
+When we define `TIME INDEX` column,
+for example `CREATE TABLE` SQL:
+
+```SQL
+CREATE TABLE system_metrics (
+    host STRING,
+    idc STRING,
+    ts TIMESTAMP NOT NULL,
+    PRIMARY KEY(host, idc),
+    TIME INDEX(ts)
+);
+```
+
+there are two special column specifications: `PRIMARY KEY` and `TIME INDEX`. The first one is used to define primary keys (or tags in some systems), and the second is to define which column is the special timestamp column.
+
+For `TIMESTAMP` column, a simplification is to change the grammar to:
+
+```SQL
+ts TIMESTAMP TIME INDEX,
+```
+
+By doing so, it
+
+- need not to write `NOT NULL`, this is implied by `TIME INDEX` keywords
+- need not to specify `TIME INDEX` in another line
+
+GreptimeDB now supports both of these two grammar rules. Users can choose one of them according to their preference.
+
+### Issue #602 (**Help Wanted**)
+
+Link:  https://github.com/GreptimeTeam/greptimedb/issues/602
+
+Issue description:
+
+Add system tables for inner metrics.
+It's good to have a metrics table like the one in [Databend](https://databend.rs/doc/sql-reference/system-tables/system-metrics) or [ClickHouse](https://clickhouse.com/docs/en/operations/system-tables/metrics/). Users can easily "select" on the metrics table to see DB's important running status. With the metrics table, we can integrate other visualization tools or dashboards that support SQL like Grafana, to export some meaningful time series data.
```

Comment:
With the metrics table, we can integrate other visualization tools or dashboards that support SQL like Grafana, to export some meaningful time series data.  
```please review this sentence above, feel like the sub is missing

Reply:
sub is unclear